src/FEValues/common_values.jl

```diff
@@ -53,8 +55,17 @@ Return the number of quadrature points in `fv`s quadrature for the current
 """
 getnquadpoints(fe::FaceValues) = getnquadpoints(fe.qr, fe.current_face[])
 
+"""
+    getnquadpoints(iv::InterfaceValues)
+
+Return double the number of quadrature points in `iv`s current [`FaceValues`](@ref)' quadrature for the current
+(most recently [`reinit!`](@ref)ed) interface.
+"""
+getnquadpoints(iv::InterfaceValues) = 2 * getnquadpoints(iv.face_values.qr, iv.face_values.current_face[])
```

Comment:
Just noticed this one (haven't looked at the rest of the code), but why is this one doubled? Presumably the idea is that both facevalues share a quadrature rule?